src/views/tables/PositionsTable.tsx

```diff
@@ -552,3 +547,13 @@ Styled.PositionSide = styled.span`
 Styled.Icon = styled(Icon)`
   font-size: 3em;
 `;
+
+Styled.MarketTableCell = styled(MarketTableCell)`
+  > * {
+    color: var(--color-text-2);
+
+    > :nth-child(2) {
```

Comment:
ehh I don't love styling by the count of children (I feel like it's sorta brittle) - also unclear what we're applying this text color to. Is there another way we can do this?